### PR TITLE
[FIX] Show Chickens on Visit

### PR DESCRIPTION
**Pull request by j923sneaks**
# Description

This PR shows chickens (normal and mutants) when visiting a farm. It also moves the type and positions.

Fixes #issue N/A

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)

# How Has This Been Tested?

Manual Testing

Visit on Testnet

chickens-visit-testnet

Regression test on Local due to refactor

chickens-visit-local-regression

# Checklist:

- [x] Title of the PR is relevant and is prefixed with [FEAT], [CHORE] or [FIX]
- [x] Screenshot if it includes any UI changes
- [x] I have read the contributing guidelines and agree to the T&Cs
- [x] I have performed a self-review of my own code
- [x] My changes generate no new warnings
- [x] New and existing unit tests pass locally with my changes
- [x] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

**Comment by ahh1539** [comment]
Would love to see peoples chickens! Great idea!

File: src/features/game/lib/constants.ts [hunk 1]
```diff
@@ -1,13 +1,31 @@
...
-import { GameState, Inventory } from "../types/game";
+import { ChickenPosition, GameState, Inventory } from "../types/game";
 
 export const GRID_WIDTH_PX = 42;
 export const CHICKEN_TIME_TO_EGG = 1000 * 60 * 60 * 24 * 2; // 48 hours
 export const MUTANT_CHICKEN_BOOST_AMOUNT = 0.1;
 
 export const POPOVER_TIME_MS = 1000;
 
+export const CHICKEN_POSITIONS: ChickenPosition[] = [
```
Comment (tiffanydys) [inline src/features/game/lib/constants.ts:11]: nice job moving it to `constants.ts` looks much cleaner!

**Review by tiffanydys — APPROVED** [review]
Works like a charm, nice job 😄

**Review by spencerdezartsmith — APPROVED** [review]
Excellent initiative @j923sneaks! Love your work 🎉 🐔

**Pull request merged**